curvefs/src/client/rpcclient/base_client.cpp

```diff
@@ -148,6 +148,60 @@ void MDSBaseClient::AllocS3ChunkId(uint32_t fsId,
     curvefs::mds::MdsService_Stub stub(channel);
     stub.AllocateS3Chunk(cntl, &request, response, nullptr);
 }
+
+// TODO(all): do we really need pass `fsId` all the time?
+//            each curve-fuse process only mount one filesystem
```

Comment:
fsid is used to mark curve-fuse. The block group is uniquely identified by the owner, 
the representation in the s3 scene is fsid.

Reply:
I mean we can store fsid as a global variable, so that, we can just use this variable, and don't have to pass it as a parameter all the time and everywhere.